examples/single_calculations/example_dft_md_reftraj.py

```diff
@@ -0,0 +1,179 @@
+###############################################################################
```

Comment:
Maybe this example is unnecessary as this run is also included in the `example_base_md_reftraj_restart.py` file.

Reply:
I agree I will remove it

Reply:
done in [48f41b7](https://github.com/aiidateam/aiida-cp2k/pull/207/commits/48f41b7503c483ba2bc193274d451ec8a0311bb8)